include/pros/distance.h

```diff
@@ -0,0 +1,109 @@
+/**
+ * \file pros/distance.h
+ *
+ * Contains prototypes for functions related to the VEX Distance sensor.
+ *
+ * Visit https://pros.cs.purdue.edu/v5/tutorials/topical/distance.html to learn
+ * more.
+ *
+ * This file should not be modified by users, since it gets replaced whenever
+ * a kernel upgrade occurs.
+ *
+ * Copyright (c) 2017-2020, Purdue University ACM SIGBots.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#ifndef _PROS_DISTANCE_H_
+#define _PROS_DISTANCE_H_
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#ifdef __cplusplus
+extern "C" {
+namespace pros {
+namespace c {
+#endif
+
+/**
+ * Get the currently measured distance from the sensor in mm
+ *
+ * This function uses the following values of errno when an error state is
+ * reached:
+ * ENXIO - The given value is not within the range of V5 ports (1-21).
+ * ENODEV - The port cannot be configured as an Distance Sensor
+ *
+ * \param  port
+ * 				 The V5 Distance Sensor port number from 1-21
+ * \return The distance value or PROS_ERR_F if the operation failed, setting
+ * errno.
+ */
+
+int32_t distance_get(uint8_t port);
+
+/**
+ * Get the confidence in the distance reading
+ *
+ * This is a value that has a range of 0 to 63. 63 means high confidence,
+ * lower values imply less confidence. Confidence is only available
+ * when distance is > 200mm.
```

Comment:
do we know what it returns if distance is > 200mm? if it's something detectable (like, not just a garbage value), maybe we should return PROS_ERR in that case and set something like EDOM (or a more appropriate value of errno)

Reply:
When distance is < 200mm the sensor returns 10 for confidence. However, the device also returns 10 during normal use.